dev/com.ibm.ws.logging_fat/publish/servers/com.ibm.ws.logging.stackjoiner/server.xml

```diff
@@ -0,0 +1,22 @@
+<!--
+    Copyright (c) 2020 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<server description="Server for testing Liberty logging in a server">
```

Comment:
Don't you need a server.env file as well for this server in order to enable the new function?

Reply:
fixed in [96faaf5](https://github.com/OpenLiberty/open-liberty/pull/16276/commits/96faaf592d1ddeb0455f5be1d4ad792995503d05)